cogs/playerstats.py

```diff
@@ -105,16 +105,33 @@ async def assists(self, interaction: discord.Interaction, competition: str = 'pl
         name="injuries",
         description="Get the list of injuries and details about them"
     )
-    async def injuries(self, interaction: discord.Interaction, team: str = 'Arsenal'):
-        if team != "Arsenal":
-            return await interaction.response.send_message("Sorry, currently only Arsenal injuries can be seen.  Stay tuned for other teams")
-        injuries = getInjuries(team)
-        embed = discord.Embed(
-            color=0x9C824A,
-            title=f"Injuries for {team.title()}"
-        )
-        for i in injuries:
-            embed.add_field(name=f'**{i["Player"]}**', value=f"> Reason: *{i['Reason']}*\n> Further Detail: *{i['FurtherDetail']}*\n> Potential Return: *{i['PotentialReturn']}*\n> Condition: *{i['Condition']}*\n> Status: *{i['Status']}*", inline=False)
+    async def injuries(self, interaction: discord.Interaction):
+        injuries_url = "https://www.premierinjuries.com/injury-table.php"
+        headers = {
+            "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.75 Safari/537.36",
+            "X-Requested-With": "XMLHttpRequest"
+        }
+        response = requests.get(injuries_url, headers=headers).text
+        parsed_page = BeautifulSoup(response, features="lxml")
+        team_table = parsed_page.find_all('tr', attrs={'class': "player-row team_1"})
+        injured_players = [i.text for i in team_table]
+
+        embed = discord.Embed(color=0x9C824A)
+        embed.set_footer(text=f"Data is sourced from {injuries_url}")
+
+        for player in injured_players:

```

Comment:
Can you add a comment indicating what the `player` object looks like? Makes it easier for others to understand the code

Reply:
Will see what I can do as its quite large (that's what she...)
![image](https://github.com/AndyReifman/ArseneWenger/assets/38725593/ccb81f36-448e-4866-8875-3d7d109e51e2)

Reply:
just do the first player.

Reply:
Added